.github/workflows/release.yaml

```diff
@@ -0,0 +1,93 @@
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Pushing Packages"
+              cd /work
+              export AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1)
+              autonomy init --reset --author $AUTHOR --ipfs --remote
+              autonomy push-all
+  publish-images:
+    name: Publish Docker Images
+    runs-on: ${{ matrix.os }}
+    needs:
+      - "publish-packages"
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+    env:
+        DOCKER_USER: ${{secrets.DOCKER_USER}}
+        DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up tag and vars
+        uses: addnab/docker-run-action@v3
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            run: |
+              echo "Setting Tag Images"
+              cd /work
+              apt-get update && apt-get install git -y || exit 1
+              export TAG=$(git describe --exact-match --tags $(git rev-parse HEAD)) || exit 1
+              if [ $? -eq 0 ]; then
+                  export TAG=`echo $TAG | sed 's/^v//'`
+              else
+                  echo "You are not on a tagged branch"
+                  exit 1
+              fi
+              echo VERSION=$TAG> env.sh
+              echo AUTHOR=$(grep 'service' packages/packages.json | awk -F/ '{print $2}' | head -1) >> env.sh
+              echo SERVICE=$(grep 'service' packages/packages.json | awk -F/ '{print $3}' | head -1) >> env.sh
+              echo DEFAULT_IMAGE_TAG=$(cat packages/packages.json | grep agent | awk -F: '{print $2}' | tr -d '", ') >> env.sh
+              cat env.sh
+
+      - uses: addnab/docker-run-action@v3
+        name: Build Images
+        with:
+            image: valory/open-autonomy-user:latest
+            options: -v ${{ github.workspace }}:/work
+            shell: bash
+            run: |
+              echo "Building Docker Images"
+              cd /work
+              source env.sh || exit 1
+              echo "Building images for $AUTHOR for service $SERVICE"
+              autonomy fetch $AUTHOR/$SERVICE --service --local || exit 1
+              cd $SERVICE || exit 1
+              autonomy build-image || exit 1
+              autonomy build-image --version $VERSION || exit 1
+
+
+      - name: Docker login
+        run: |
+            echo  $DOCKER_PASSWORD | docker login -u $DOCKER_USER --password-stdin 
+      - name: Docker Push
+        run: |
+          source env.sh
+          echo "Pushing $DOCKER_USER/$SERVICE:$VERSION"
+          echo "Pushing $DOCKER_USER/$SERVICE:$DEFAULT_IMAGE_TAG"
+          docker push $DOCKER_USER/oar-$SERVICE:$VERSION
+          docker push $DOCKER_USER/oar-$SERVICE:$DEFAULT_IMAGE_TAG
```

Comment:
we have two services in the repo, only one of them will get a tag. Should both of them get tags?

Reply:
Correct me if im wrong, but the oar image is actuall the same for both services

Reply:
so this hsould be the same name

Reply:
![image](https://user-images.githubusercontent.com/35799987/214554730-4ce39cb0-536e-441f-87bf-5cbf7bb7dfe8.png)

@jose thanks for confirming

Reply:
Otheerwise, we will need to put in a function, and then do the function for each service. 

I *think* we can do loops in actions,